Cargo.toml

```diff
@@ -10,7 +10,7 @@ license = "GPL-3.0"
 edition = "2018"
 
 [features]
-default = ['gnt-driver', 'gftp/bin']
+default = ['zksync-driver', 'gftp/bin']
```

Comment:
I'd say we should include both in the default build.

Reply:
Great idea! updated